### PR TITLE
#216 Additional options for update/revert apply operations

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://github.com/wildfly-extras/prospero/issues/216

### CODE REVIEW

File: prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/UpdateCommand.java [hunk 1]
```diff
@@ -179,6 +179,12 @@ public static class ApplyCommand extends AbstractCommand {
         @CommandLine.Option(names = CliConstants.CANDIDATE_DIR, required = true)
         Path candidateDir;
 
+        @CommandLine.Option(names = CliConstants.REMOVE)
+        boolean remove;
+
+        @CommandLine.Option(names = CliConstants.SKIP_CONFLICTS)
```
Comment (spyrkob) [inline prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/UpdateCommand.java:185]: turns out this part is already implemented (as `yes` parameter). Sorry my mistake I forgot it was already done in a different issue. Could you remove the `skipConflicts` part please?

**Review by spyrkob — CHANGES_REQUESTED** [review]
Thanks @parsharma looks good. A few comments below.

Could you also add unit tests for removing the candidate and a usage description for `--rm` in UsageMessages.properties please?

**Comment by parsharma** [comment]
Thanks for the review comments. I have updated the code.

**Review by spyrkob — APPROVED** [review]
LGTM, thanks @parsharma

**Comment by spyrkob** [comment]
@parsharma sorry, I'm holding on with merging this until we're ready to move to 1.1.x. I'll resolve the conflicts when it's ready

**Comment by spyrkob** [comment]
Superseded by #456.

Thanks @parsharma I rebased your PR and will merge it in new PR. Sorry for delay!

**Pull request closed without merging**